doc/user/data/examples/create_table/example_user_populated_table.yml

```diff
@@ -0,0 +1,87 @@
+- name: "syntax"
```

Comment:
Can skip. Just converting existing create table railroad diagrams and various tables.